ERROR - 2022-10-11 18:57:13 --> mysqli_sql_exception: Incorrect table definition; there can be only one auto column and it must be defined as a key in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#16 {main}
ERROR - 2022-10-11 18:59:43 --> mysqli_sql_exception: Incorrect table definition; there can be only one auto column and it must be defined as a key in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#16 {main}
ERROR - 2022-10-11 19:02:00 --> mysqli_sql_exception: Incorrect table definition; there can be only one auto column and it must be defined as a key in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#16 {main}
ERROR - 2022-10-11 19:03:24 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#16 {main}
ERROR - 2022-10-11 19:05:40 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:07:24 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:15:55 --> mysqli_sql_exception: Incorrect table definition; there can be only one auto column and it must be defined as a key in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(55): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:16:26 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:17:15 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:17:27 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:18:14 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:19:49 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:22:51 --> mysqli_sql_exception: Table 'usuarios' already exists in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(53): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:23:42 --> mysqli_sql_exception: Table 'usuarios' already exists in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(53): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:24:56 --> mysqli_sql_exception: Table 'usuarios' already exists in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(53): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:29:03 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#16 {main}
ERROR - 2022-10-11 19:31:03 --> mysqli_sql_exception: Table 'puntaje' already exists in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233225_Puntaje.php(26): CodeIgniter\Database\Forge->createTable('puntaje')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Puntaje->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:31:24 --> mysqli_sql_exception: Table 'puntaje' already exists in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233225_Puntaje.php(28): CodeIgniter\Database\Forge->createTable('puntaje')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Puntaje->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:31:40 --> mysqli_sql_exception: Table 'puntaje' already exists in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233225_Puntaje.php(28): CodeIgniter\Database\Forge->createTable('puntaje')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Puntaje->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:32:59 --> mysqli_sql_exception: Can't create table `sateg`.`puntaje` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233225_Puntaje.php(28): CodeIgniter\Database\Forge->createTable('puntaje')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Puntaje->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#16 {main}
ERROR - 2022-10-11 19:36:15 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:36:26 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(112): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#16 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#17 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-10-11 19:37:15 --> mysqli_sql_exception: Can't create table `sateg`.`usuarios` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\sateg\sistema-de-puntos\app\Database\Migrations\2022-10-11-233131_Usuarios.php(54): CodeIgniter\Database\Forge->createTable('usuarios')
#5 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(868): App\Database\Migrations\Usuarios->up()
#6 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(900): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\sateg\sistema-de-puntos\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#15 C:\xampp\htdocs\sateg\sistema-de-puntos\spark(98): CodeIgniter\CLI\Console->run()
#16 {main}
